### PR TITLE
[Fixed #8008] Fixed DApps permission regression

### LINKED ISSUE #8008: Error "No protocol method IDeref.-deref defined for type undefined:" and crash if open DApp permissions
### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: Error and crash if open DApp permissions
```
04-22 09:48:15.835 12036 12081 E ReactNativeJS: Error: No protocol method IDeref.-deref defined for type undefined: 
04-22 09:48:15.835 12036 12081 E ReactNativeJS: 
04-22 09:48:15.835 12036 12081 E ReactNativeJS: This error is located at:
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in dapps-permissions
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in reagent66
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in reagent65
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in r
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in s
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in u
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in C
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in n
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in E
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in n
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in H
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in L
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in h
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in v
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in h
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in _
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in N
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in s
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in l
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in n
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in n
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in _
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in h
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in N
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in s
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in u
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in C
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in n
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in E
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in n
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in H
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in L
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in h
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in v
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in h
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in _
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in N
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in s
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in c
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in h
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in N
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in reagent2
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in root
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in reagent1
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in RCTView
04-22 09:48:15.835 12036 12081 E ReactNativeJS:     in c

```

#### Expected behavior

Dapp permission screen is opened 

#### Actual behavior

Error "No protocol method IDeref.-deref defined for type undefined:" and crash

### Reproduction

[comment]: # (Describe how we can replicate the bug step by step.)

- Open Status and create new account
- Profile -> DApp permissions

### Additional Information

[comment]: # (Please do your best to fill this out.)
- Status version: nightly April 22 - 0.12.0(2019042202)
- iOS 12.2, real iphone 7 plus 
- Android 9, real Pixel xl

#### Logs

[Status.log](https://github.com/status-im/status-react/files/3102654/Status.log)

### PR DESCRIPTION
**Pull request by jeluard**
fixes #8008

### Summary

Fixed regression breaking DApps permission screen.

status: ready

### CODE REVIEW

File: src/status_im/contact/subs.cljs [hunk 1]
```diff
@@ -128,3 +128,28 @@
                acc))
            {}
            contacts)))
+
+(re-frame/reg-sub
+ ::dapps
+ (fn [db]
+   (:contacts/dapps db)))
```
Comment (goranjovic) [inline src/status_im/contact/subs.cljs:135]: minor nitpick - is there any value in having this as a local sub as opposed to just nesting `(:contacts/dapps db)` in the calling code?

**Review by goranjovic — APPROVED** [review]

File: src/status_im/contact/subs.cljs (hunk 1 above)
Comment (jeluard) [reply src/status_im/contact/subs.cljs:135]: No personal opinion on this. I just reverted some previously deleted code.

**Comment by flexsurfer** [comment]
we don't have dapps in contacts anymore, we removed them when implemented dap.ps discovery dapp, so i prepared the proper fix for this issue https://github.com/status-im/status-react/pull/8017, thanks

**Comment by jeluard** [comment]
Closing in favour of #8017

**Pull request closed without merging**